app/scripts/background.js

```diff
@@ -623,6 +636,10 @@ export async function loadStateFromPersistence() {
     });
   });
 
+  if (!preMigrationVersionedData?.data && !preMigrationVersionedData?.meta) {
+    preMigrationVersionedData = migrator.generateInitialState(firstTimeState);
```

Comment:
A bit of a paranoid change... previously if we didn't have _any_ `preMigrationVersionedData` (it was `undefined`) we would use `migrator.generateInitialState(firstTimeState)`.

Now we also use `migrator.generateInitialState(firstTimeState)` in the case where `data` and `meta` are both not set.

I don't know why that would ever happen, but Dan thought it might and added this logic in his original PR.